### PR TITLE
Removes useless uniforms

### PR DESCRIPTION
**Pull request by alsoandanswer**
basically what it says in the title.
Just deletes some useless uniforms because every time I opened the .dmi I would get a brain hemorrhage.

Most of these aren't obtainable through the loadout anyway and would require admin fuckery/cargo/mainthunting, so there wouldn't be much frontside changes to people's loadouts

Either that or they were some dude's ancient hardcoded custom item or a part of some adminspawn thing that we don't even use anyway

firefox_sqUXkhLt66
like what the fuck is this shit???

Furthermore some can be already be replicated using customizable things in the loadout

And some of these were just flat out piss ugly so

**note : this uses the skirt PR (#8938 )  as well as the sweater PR (#8937 ) as a base so I wouldn't end up with merge errors - i suggest merging them first**

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/datums/outfits/outfit_admin.dm [hunk 1]
```diff
@@ -338,7 +338,6 @@
 /datum/outfit/admin/nt/fib
 	name = "FIB Agent"
 
-	uniform = /obj/item/clothing/under/rank/fib
```
Comment (Alberyk) [inline code/datums/outfits/outfit_admin.dm:341]: This is used by the ccia, should stay.

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -99,12 +78,6 @@
 	worn_state = "centcom"
 	displays_id = 0
 
-/obj/item/clothing/under/rank/fib
```
Comment (Alberyk) [inline code/modules/clothing/under/miscellaneous.dm:102]: This is used by the ccia, should stay.

File: code/modules/clothing/under/miscellaneous.dm [hunk 3]
```diff
@@ -160,35 +133,13 @@
...
-	item_state = "johnny"
-
 /obj/item/clothing/under/rainbow
 	name = "rainbow"
 	desc = "rainbow"
 	icon_state = "rainbow"
 	item_state = "rainbow"
 	worn_state = "rainbow"
 
-/obj/item/clothing/under/psysuit
```
Comment (Alberyk) [inline code/modules/clothing/under/miscellaneous.dm:177]: Used by antags.

File: code/modules/clothing/under/miscellaneous.dm [hunk 4]
```diff
@@ -160,35 +133,13 @@
...
-	name = "dark undersuit"
-	desc = "A thick, layered grey undersuit lined with power cables. Feels a little like wearing an electrical storm."
-	icon_state = "psysuit"
-	item_state = "bl_suit"
-	worn_state = "psysuit"
-	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|FEET|ARMS|HANDS
-
-/obj/item/clothing/under/gentlesuit
```
Comment (Alberyk) [inline code/modules/clothing/under/miscellaneous.dm:185]: Used by antags.

File: code/modules/clothing/under/miscellaneous.dm [hunk 5]
```diff
@@ -231,35 +182,6 @@
...
 	worn_state = "red_suit"
 
-/obj/item/clothing/under/overalls
-	name = "laborer's overalls"
-	desc = "A set of durable overalls for getting the job done."
-	icon_state = "overalls"
-	item_state = "lb_suit"
-	worn_state = "overalls"
-
-/obj/item/clothing/under/pirate
```
Comment (Alberyk) [inline code/modules/clothing/under/miscellaneous.dm:241]: Used by antags.

File: code/modules/clothing/under/miscellaneous.dm [hunk 6]
```diff
@@ -231,35 +182,6 @@
 	item_state = "r_suit"
 	worn_state = "red_suit"
 
-/obj/item/clothing/under/overalls
```
Comment (Alberyk) [inline code/modules/clothing/under/miscellaneous.dm:234]: Used by antags.

File: code/modules/clothing/under/miscellaneous.dm (hunk 5 above)
Comment (alsoandanswer) [reply code/modules/clothing/under/miscellaneous.dm:241]: Not readding this. We've got better options

File: code/modules/clothing/under/miscellaneous.dm (hunk 4 above)
Comment (alsoandanswer) [reply code/modules/clothing/under/miscellaneous.dm:185]: Too similar to existing other suits already.

File: code/datums/outfits/outfit_admin.dm (hunk 1 above)
Comment (alsoandanswer) [reply code/datums/outfits/outfit_admin.dm:341]: Readded.

File: code/modules/clothing/under/miscellaneous.dm (hunk 3 above)
Comment (alsoandanswer) [reply code/modules/clothing/under/miscellaneous.dm:177]: Readded, since it's unique

File: code/modules/clothing/under/miscellaneous.dm (hunk 6 above)
Comment (alsoandanswer) [reply code/modules/clothing/under/miscellaneous.dm:234]: Readded

**Pull request merged**